README.md

```diff
@@ -2,20 +2,17 @@
 At Hackney, we have created the NuGet Package to prevent the duplication of common code when implementing our APIs.
 Hence this NuGet package will store the common code that can then be used in the relevant projects. 
 
-#### CircleCI Pipeline - Versioning
-At present the pipeline does not automatically update the package version number.
+#### GitHub Actions Pipeline - Versioning
+The pipeline automatically updates the package version number using [GitVersion](https://gitversion.net/).
 
-**This means that for the NuGet Push command to work when code is merged to the release branch 
-you must change the version number in the .csproj file according to the type of change you are making.**
+Version numbers use the following format:
 
-The new version number should use the following format.
+Any specific version number follows the form Major.Minor.Patch[-Suffix], where the components have the following meanings:
 
-    A specific version number is in the form Major.Minor.Patch[-Suffix], where the components have the following meanings:
-
-    Major: Breaking changes
-    Minor: New features, but backward compatible
-    Patch: Backwards compatible bug fixes only
-    Suffix (optional): a hyphen followed by a string denoting a pre-release version
+* *Major*: Breaking changes
+* *Minor*: New features, but backward compatible
+* *Patch*: Backwards compatible bug fixes only
+* *Suffix (optional)*: a hyphen followed by a string denoting a pre-release version
 
 ## Using the package
 For full details on how to use the package(s) within this repository please read 

```

Comment:
The features section below this point should be updated to indicate that the existing Hackney.Core package has now been split up into individual packages and is now obsolete. It should also refer to the specific packages for each functional area.